auth/auth_test.go

```diff
@@ -880,6 +884,20 @@ func TestVerifyIDTokenAndCheckRevokedError(t *testing.T) {
 	}
 }
 
+func TestVerifyIDTokenAndCheckDisabledError(t *testing.T) {
```

Comment:
In other languages we also added integration tests to verify this functionality. We might want to do the same here.

Reply:
Where should I add integration tests?  I'm trying to find other examples of integration tests that use VerifyIDToken and/or checkRevokedOrDisabled, and having a bit of a hard time doing so.